esmvaltool/recipes/recipe_hyintex.yml

```diff
@@ -0,0 +1,130 @@
+# recipe_hyint.yml
```

Comment:
This looks like a duplicate of recipe_hyint.yml. Is this file here intentionally?

Reply:
Intentionally. This is a special case of the main hyint recipe, in this case reading climate extreme indices produced by the climdex library with the extremeEvents recipe/namelist/diagnostics (which should then work one in line with the other one). We are awaiting further progress on the porting of the extremeEvents diagnostics into v2, meanwhile attempting to produce the indices ourself. The climdex library is not able to deal with netcdf files pre-processed by the v2 preprocessor. I will (re-?)open an issue on the climdex library.

Reply:
Ok, thank you for the explanation.

Reply:
I removed this recipe until it can be used to avoid confusion.